db/migrate/20170822173332_add_swags_to_sponsors.rb

```diff
@@ -0,0 +1,13 @@
+class AddSwagsToSponsors < ActiveRecord::Migration
+  def change
+    add_column :sponsors, :paid, :boolean, default: false
+    add_column :sponsors, :has_swag, :boolean, default: false
+    add_column :sponsors, :swag_received, :boolean
+    add_column :sponsors, :address, :string
+    add_column :sponsors, :vat, :string
+    add_column :sponsors, :has_banner, :boolean, default: false
+    add_column :sponsors, :swag, :text
+    add_column :sponsors, :courier_info, :text
```

Comment:
I don't like the variables we are introducing for this. You first need to come up with your plan, present it, and then implement it, after we discuss it and agree on it. You should have done that for the new attributes and variables you are introducing. 
The courier_info for a sponsor means very little. The swag transportation info for a sponsor means much more. Please come up with meaningful attributes, and link me to your mockups and workflows document.